src/ResourceManager/DataMigration/Commands.DataMigration/help/New-AzureRmDataMigrationSqlServerSqlDbSelectedDB.md

```diff
@@ -1,7 +1,7 @@
-﻿---
+---
```

Comment:
Shouldn't this be removed in favor of the new cmdlet?

Reply:
I wasn't sure if it was okay to remove it or not (from backward compatibility point of view.).
I kept it and was planning to remove it in later versions.